crates/sovereign-sdk/module-system/sov-modules-stf-blueprint/src/lib.rs

```diff
@@ -748,6 +748,12 @@ where
                     da_block_header = da_block_headers_iter.next().unwrap();
                 }
 
+                assert_eq!(
+                    soft_confirmation.l2_height(),
```

Comment:
seqeuncer commitment start and end are also checked because of the way I added these. see lines above and below